core-web/libs/edit-content/src/lib/fields/dot-edit-content-category-field/components/dot-category-field-search/dot-category-field-search.component.html

```diff
@@ -0,0 +1,21 @@
+<div class="p-inputgroup w-full">
+    <input
+        #searchInput
+        class="category-field__search-input"
+        pInputText
+        placeholder="{{ 'edit.content.category-field.search.input.placeholder' | dm }}"
```

Comment:
[placeholder]="'edit.content.category-field.search.input.placeholder' | dm"

Is the same, but it is more common to see data binding like this

Reply:
changed.